src/classes/User.js

```diff
@@ -2,29 +2,27 @@ class User {
   constructor(userData) {
     this.name = userData.name;
     this.id = userData.id;
-    this.pantry = userData.pantry;
-    this.savedRecipes = [];
+    this.recipesToCook = userData.recipesToCook;
   }
 
   toggleSaveRecipe(recipe) {
     if (recipe.saved) {
-      this.savedRecipes.forEach((element, index) => {
-        element.id === recipe.id ? this.savedRecipes.splice(index, 1) : null;
+      this.recipesToCook.forEach((element, index) => {
+        element.id === recipe.id ? this.recipesToCook.splice(index, 1) : null;
       });
     } else {
-      this.savedRecipes.unshift(recipe);
+      this.recipesToCook.unshift(recipe);
     }
     recipe.toggleSave();
   }
```

Comment:
We'll have to refactor this method since the saved property isn't getting passed to the server.

Reply:
See comment below but I think all the logic in this method can be replaced by that conditional.

Reply:
Followed by a new get and refresh of the user.recipesToCook array